### PR TITLE
add other language drop down link to AutoGen website

### PR DESCRIPTION
**Pull request by LittleLittleCloud**
## Why are these changes needed?
This PR adds a 'OtherLanguage' drop down link plus link to AutoGen.Net document website

image

#924 

## Checks

- [ ] I've included any doc changes needed for https://microsoft.github.io/autogen/. See https://microsoft.github.io/autogen/docs/Contribute#documentation to build and test documentation locally.
- [ ] I've added tests (if relevant) corresponding to the changes introduced in this PR.
- [ ] I've made sure all auto checks have passed.

### CODE REVIEW

File: website/process_notebooks.py [hunk 1]
```diff
@@ -43,7 +43,7 @@ def notebooks_target_dir(website_directory: Path) -> Path:
 
 
 def extract_yaml_from_notebook(notebook: Path) -> typing.Optional[typing.Dict]:
-    with open(notebook, "r") as f:
+    with open(notebook, "r", encoding='utf-8') as f:
```
Comment (LittleLittleCloud) [inline website/process_notebooks.py:46]: Add `encoding` to make it run on windows

**Comment by jackgerrits** [comment]
I wonder if this is the best place to expose this. I think it important to prominently link to this but I am worried about cluttering the navbar (which I am hoping to clean up soon)

**Comment by LittleLittleCloud** [comment]
> I wonder if this is the best place to expose this. I think it important to prominently link to this but I am worried about cluttering the navbar (which I am hoping to clean up soon)

If the nav bar is already too crowded, how about putting it under resource tab?

aka
Resource->other languages->dotnet

**Review by jackgerrits — APPROVED** [review]
It's good to highlight this and we can always move it later if we need to.

**Pull request merged**